packages/dx-react-grid/docs/guides/export.md

```diff
@@ -0,0 +1,94 @@
+# React Grid - Export
+
+The Grid component supports exporting data in Excel format.
+
+## Related Plugins
+
+- [Toolbar](../reference/toolbar.md) - renders the Grid Toolbar
+- [ExportPanel](../reference/export-panel.md) - renders the Export Panel
```

Comment:
ExportPanel renders only export button

Reply:
Anything that renders into Toolbar is what we refer to as panel

Reply:
Sound like we render other panel)